deploy/kube/mixer.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: mixer
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: mixer
+      annotations:
+        prometheus_port: "42422"
```

Comment:
Using an extra annotation is not the correct way to discover the port. Correct way is to expose the 42422 port in the mixer service, then use standard prometheus meta labels in the prometheus job:

"The service role discovers a target for each service port for each service. This is generally useful for blackbox monitoring of a service. The address will be set to the Kubernetes DNS name of the service and respective service port.

Available meta labels:

__meta_kubernetes_namespace: The namespace of the service object.
__meta_kubernetes_service_name: The name of the service object.
__meta_kubernetes_service_label_<labelname>: The label of the service object.
__meta_kubernetes_service_annotation_<annotationname>: The annotation of the service object.
__meta_kubernetes_service_port_name: Name of the service port for the target.
__meta_kubernetes_service_port_number: Number of the service port for the target.
__meta_kubernetes_service_port_portocol: Protocol of the service port for the target.
"